next.config.js

```diff
@@ -256,6 +256,10 @@ const nextConfig = {
         source: `/${redirectBase}learn/package-references`,
         destination: `/${redirectBase}learn/development-tutorials/source-code-dependencies/package-references`,
       },
+      {
+        source: `/${redirectBase}learn/customizing-formatting`,
+        destination: `/${redirectBase}learn/development-tutorials/source-code-dependencies/customizing-formatting`,
+      },
```

Comment:
Will this add a  new left nav entry as well?

Reply:
Yes, decided to have a separate left nav entry under source code and dependencies